docs/src/modules/components/AppNavDrawer.js

```diff
@@ -194,6 +181,8 @@ const AppNavPaperComponent = styled('div')(() => {
   return {
     width: 'var(--MuiDocs-navDrawer-width)',
     boxShadow: 'none',
+    border: '0 !important', // TODO add a Paper slot
+    overflowY: 'unset !important', // TODO add a Paper slot
```

Comment:
@DiegoAndai This makes me think of https://github.com/mui/material-ui/pull/38021#issuecomment-1640604578. I think this illustrates the value of exporting all the slots in Material UI.

I would want to do this:

```jsx
import { SwipeableDrawer, DrawerPaper } from "@mui/material"

const AppNavPaperComponent = styled(DrawerPaper)({});

<SwipeableDrawer slots={{ paper: AppNavPaperComponent }} />
```

But I can't.

Or even better, relying on composition (slots feel like a drain, the fewer we have, the better):

```jsx
import { SwipeableDrawer, DrawerPaper } from "@mui/material"

const AppNavPaperComponent = styled(DrawerPaper)({});

<SwipeableDrawer>
  <AppNavPaperComponent>
```

Reply:
I'll resolve this thread given it's tangential/not directly related to the PR but y'all can keep debating!